packages/sdk/src/swap/assets.ts

```diff
@@ -68,3 +68,27 @@ export const btc$: (network: ChainflipNetwork) => AssetData = (network) => ({
   minimumDepositAmount: getMinimumDepositAmount(network, Assets.BTC),
   minimumSwapAmount: getMinimumSwapAmount(network, Assets.BTC),
 });
+
+export const arbeth$: (network: ChainflipNetwork) => AssetData = (network) => ({
+  id: Assets.ARBETH,
+  chain: Chains.Arbitrum,
+  contractAddress: undefined,
+  decimals: assetDecimals[Assets.ARBETH],
+  name: 'Arbitrum',
+  symbol: 'ARBETH',
```

Comment:
think i woul use `ETH` and `USDC` as symbol

Reply:
but we can still change it later

Reply:
Agreed, it's similar to our `swapToken`/`callToken` stuff. It makes sense with the context we have but for users of the SDK without the context, it seems like an arbitrary decision.

Not a fan of this convention in general because it overloads the asset id to be a compound identifier of chain and asset ID but only for non-Ethereum EVM chains.

Reply:
we can decide on this later.. for now especially during dev it will help to have it be called ARBETH especially since that's what they're calling it on the backend. And calling this ETH would mean a bigger refactor of how we define Assets and I don't think that should be a part of this PR